cmd/util/cmd/read-protocol-state/cmd/blocks.go

```diff
@@ -171,5 +204,38 @@ func run(*cobra.Command, []string) {
 		return
 	}
 
-	log.Fatal().Msgf("missing flag, try --final or --sealed or --height or --block-id")
+	if flagExecuted {
```

Comment:
Might be nice adding an extra log line telling you not to use `final` `sealed` and `executed` at the same time.